crystal-map-connector/src/test/kotlin/ch/ergon/dope/helper/Builder.kt

```diff
@@ -15,11 +17,15 @@ import ch.ergon.dope.validtype.ArrayType
 import ch.ergon.dope.validtype.BooleanType
 import ch.ergon.dope.validtype.NumberType
 import ch.ergon.dope.validtype.StringType
+import ch.ergon.dope.validtype.ValidType
 import com.schwarz.crystalapi.schema.CMField
 import com.schwarz.crystalapi.schema.CMList
+import com.schwarz.crystalapi.schema.CMType
 
 fun someBucket(name: String = "someBucket") = UnaliasedBucket(name)
 
+fun someCMType(name: String = "CMType", path: String = "") = CMField<Number>(name, path) as CMType
```

Comment:
I really like this!! Would've been useful for the tests in my PR (but I didn't think to add this myself lol)